package.json

```diff
@@ -62,6 +62,7 @@
     "prettier": "1.19.1",
     "pretty-quick": "2.0.1",
     "ts-jest": "25.2.1",
+    "ts-morph": "^7.0.0",
```

Comment:
Very useful for its simple project and in memory host features. Happy to replace if we find as easy a way with raw ts api.